### PR TITLE
feat(lsp): added trim option to make_entry.gen_from_quickfix

### PR DESCRIPTION
**Pull request by rowario**
Added option to trim results text.
image
image

### CODE REVIEW

File: lua/telescope/make_entry.lua [hunk 1]
```diff
@@ -324,6 +324,10 @@ function make_entry.gen_from_quickfix(opts)
 
     local line_info = { table.concat({ entry.lnum, entry.col }, ":"), "TelescopeResultsLineNr" }
 
+    if opts.trim_text then
```
Comment (Conni2461) [inline lua/telescope/make_entry.lua:327]: We need documentation for this option:.
Docs is autogenerated from this file: https://github.com/nvim-telescope/telescope.nvim/blob/master/lua/telescope/builtin/init.lua

`make_entry.gen_from_quickfix` is used for:
- `lsp_references`
- `lsp_definitions`
- `lsp_type_definitions`
- `lsp_implementations?
- `quickfix`
- `loclist`
- `tagstack`
- `jumplist`

**Review by Conni2461 — CHANGES_REQUESTED** [review]
If the documentation is here we can do this. Thanks :)

**Comment by rowario** [comment]
done

**Comment by Conni2461** [comment]
Thanks :)

**Pull request merged**